benchmarks/bench_plot_randomized_svd.py

```diff
@@ -118,10 +118,9 @@
 
 
 def unpickle(file):
-    fo = open(file, 'rb')
-    dict = pickle.load(fo, encoding='latin1')
-    fo.close()
-    return dict['data']
+    with open(file, 'rb') as fo:
+        dict = pickle.load(fo, encoding='latin1')
+        return dict['data']
```

Comment:
`dict` is a reserved keyword in python

You could just do a one line `return pickle.load(fo, encoding='latin1')['data']`